e2e-tests/content-strings/error-messages.js

```diff
@@ -1,7 +1,7 @@
-import { FIELD_IDS, ELIGIBILITY } from '../constants';
+import { FIELD_IDS, ELIGIBILITY, TOTAL_CONTRACT_VALUE } from '../constants';
 import formatCurrency from '../helpers/format-currency';
 
-export const MAX_COVER_AMOUNT = formatCurrency(ELIGIBILITY.MAX_COVER_AMOUNT_IN_GBP);
+export const MAX_COVER_AMOUNT = formatCurrency(TOTAL_CONTRACT_VALUE.AMOUNT_250K);
```

Comment:
Should we rename `MAX_COVER_AMOUNT` now, since there is no maximum

Reply:
changed to `THRESHOLD`